### PR TITLE
[Draft] fixes issue aligning workplane to a face

### PR DESCRIPTION
**Pull request by hvfrancesco**
this is a quick workaround for a problem introduced in the Draft select plane procedure while refactoring the code. not sure it is the best fix though.

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

**Comment by hvfrancesco** [comment]
The issue is discussed here: [https://forum.freecadweb.org/viewtopic.php?f=23&t=39189](url)

**Comment by vocx-fc** [comment]
> The issue is discussed here: [https://forum.freecadweb.org/viewtopic.php?f=23&t=39189](url)

The link doesn't point to the thread. You are supposed to fill the address in the space that says "url", between parentheses. The first space [between brackets] is for arbitrary text, like this [Align working plane to face fails in git build](https://forum.freecadweb.org/viewtopic.php?f=23&t=39189).

Although your contribution is appreciated, I think we should probably wait for the Draft author (Yorik) to look into it. He said he was going to be away from the computer for a few weeks due to travelling. As soon as he is back, he'll probably decide what is best to solve this issue.

The other managers who have merge access usually don't merge pull requests related to Draft. They will probably just wait until Yorik comes back to deal with this. So, although your pull request is good to note there is a problem, I think we should just wait for Yorik.

**Comment by hvfrancesco** [comment]
no problem, I was quite aware that this issue is probably due to a code refactoring not yet finished by Yorik. I made the PR only following the suggestion by someone on the forum, in order to let people using code from github that this could be used as temporary solution until the issue is addressed by the maintainer of the code.

**Comment by vocx-fc** [comment]
Okay. As a temporary fix it's fine. "Use this patch until the developer comes back."

File: src/Mod/Draft/DraftSelectPlane.py [hunk 1]
```diff
@@ -123,8 +112,19 @@ def Activated(self):
         self.taskd.form.buttonPrevious.clicked.connect(self.onClickPrevious)
         self.taskd.form.fieldGridSpacing.textEdited.connect(self.onSetGridSize)
         self.taskd.form.fieldGridMainLine.valueChanged.connect(self.onSetMainline)
-        self.taskd.form.fieldSnapRadius.valueChanged.connect(self.onSetSnapRadius)
+        self.taskd.form.fieldSnapRadius.valueChanged.connect(self.onSetSnapRadius)
```
Comment (luzpaz) [inline src/Mod/Draft/DraftSelectPlane.py]: please remove the trailing whitespace

**Comment by luzpaz** [comment]
Heads up: https://freecadweb.org/tracker/view.php?id=4146 is directly related to this temp fix

**Comment by elliotclee** [comment]
This patch seems to make it better, but with it I still managed to sporadically get:

Running the Python command 'Draft_SelectPlane' failed:
Traceback (most recent call last):
  File "/usr/lib64/freecad/Mod/Draft/DraftSelectPlane.py", line 98, in Activated
    self.taskd.form.buttonAlign.setIcon(QtGui.QIcon(":/icons/view-isometric.svg"))

**Comment by hvfrancesco** [comment]
> This patch seems to make it better, but with it I still managed to sporadically get:
> 
> Running the Python command 'Draft_SelectPlane' failed:
> Traceback (most recent call last):
> File "/usr/lib64/freecad/Mod/Draft/DraftSelectPlane.py", line 98, in Activated
> self.taskd.form.buttonAlign.setIcon(QtGui.QIcon(":/icons/view-isometric.svg"))

I tried a lot of different cases but I wasn't able to reproduce this issue. Is there a particular condition I could test?

**Comment by yorikvanhavre** [comment]
Thanks for the fix!

> File "/usr/lib64/freecad/Mod/Draft/DraftSelectPlane.py", line 98, in Activated
> self.taskd.form.buttonAlign.setIcon(QtGui.QIcon(":/icons/view-isometric.svg"))

This is weird, but seem unrelated to the problem fixed by this PR anyway... In any case line 98 has changed since then, so it might be best to try again with a recent version

**Pull request merged**